### ISSUE TITLE
Web port and minos/console ports should simply disallow exiting to the OS

### ISSUE THREAD

**Issue by bob-the-hamster** [ohrrpgce/ohrrpgce#1270]
Right now, if you are using a web port, and you press ESC on the file select screen (or on the title screen when loading directly into a game) then the game exists, and you are stuck on a black screen until you reload the browser tab.

I also have heard on good authority that exiting to the OS is strictly verboten for console ports, and we don't want to make game authors handle that themselves via scripts.

We should simply disallow exiting to the OS on such platforms.

**Comment by bob-the-hamster** [comment]
When implementing this, I need to take care that I don't break the feature that allows rpg files to run other rpg files, and then exit back to the original rpg file.

I'm also going to ad a `-noexit` command line argument so this behavior can be tested on desktop platforms

**Comment by rversteegen** [comment]
`-noexit` is probably not needed, because you can instead test it with `scons minos=1`.

On the other hand, a kiosk mode might have actual use.

On the other other hand (the first hand), a kiosk mode is probably not needed, because you can instead use `scons minos=1`.

**Comment by bob-the-hamster** [comment]
I want `-noexit` because I want people to be able to use it without recompiling, or downloading a different build.

I realized why I was having trouble wrapping my head around how this feature should behave. That is because the behaviour is different depending on whether or not the browse screen is skipped.

If browse is skipped then noexit forbids exiting from the title screen, the load screen, and the "game over" script command (making "game over" the same as "reset game")

However, if the browse screen is not being skipped, then title screen, load screen, and "game over" script all function normally, but the browse screen forbids exiting to the os

**Comment by bob-the-hamster** [comment]
Okay, this is implemented now. I tested it with a bunch of games, with both default title screen and scripted title screens.
I also tested it with the Megallenium 2000 collection, so I know it interacts with "run game" correctly.

Now there is no need for game authors to script special game exiting behavior for Web/Android/Console

Implemented in commit 26d1c15f44a7c07b8bde2ed29a08d04aef05fb4b
